include/avif/avif.h

```diff
@@ -538,7 +538,7 @@ AVIF_API void avifImageStealPlanes(avifImage * dstImage, avifImage * srcImage, a
 // If libavif is built with a version of libyuv offering a fast conversion between RGB and YUV for
 // the given inputs, libavif will use it. See reformat_libyuv.c for the details.
 // libyuv is faster but may have slightly less precision than built-in conversion, so avoidLibYUV
-// should be set to AVIF_TRUE when AVIF_CHROMA_UPSAMPLING_BEST_QUALITY or
+// can be set to AVIF_TRUE when AVIF_CHROMA_UPSAMPLING_BEST_QUALITY or
```

Comment:
The current wording seems to recommend that avoidLibYUV should be set to AVIF_TRUE when AVIF_CHROMA_UPSAMPLING_BEST_QUALITY or AVIF_CHROMA_DOWNSAMPLING_BEST_QUALITY is used. If what we meant is that avoidLibYUV can be set to AVIF_FALSE or AVIF_TRUE, and setting it to AVIF_TRUE when AVIF_CHROMA_UPSAMPLING_BEST_QUALITY or AVIF_CHROMA_DOWNSAMPLING_BEST_QUALITY is used produces the most precise but slowest results, then we can change "should" to "can" to clarify that. An alternative rewrite is the following:

> To get the most precise but slowest results, use AVIF_CHROMA_UPSAMPLING_BEST_QUALITY or AVIF_CHROMA_DOWNSAMPLING_BEST_QUALITY and set avoidLibYUV to AVIF_TRUE.

Reply:
Your suggestion looks good. Should it be "and" instead of "or"?

Reply:
Thank you for confirming I understood the comment correctly. In the interest of time, I am going to merge this pull request as is, because the "should" to "can" change also clarifies the comment.

Regarding the alternative that I suggested:

> To get the most precise but slowest results, use AVIF_CHROMA_UPSAMPLING_BEST_QUALITY or AVIF_CHROMA_DOWNSAMPLING_BEST_QUALITY and set avoidLibYUV to AVIF_TRUE.

I used "AVIF_CHROMA_UPSAMPLING_BEST_QUALITY or AVIF_CHROMA_DOWNSAMPLING_BEST_QUALITY" because "or" was used in the original comment. I guess I thought "or" was okay because an `avifRGBImage` struct is typically only used for either YUV-to-RGB or RGB-to-YUV conversion, so we only need to set either `chromaUpsampling` or `chromaDownsampling`. It is harmless but not necessary to set both.

Perhaps we can clarify the alternative as follows:

> To get the most precise but slowest results, set chromaUpsampling to AVIF_CHROMA_UPSAMPLING_BEST_QUALITY (for avifImageYUVToRGB) or set chromaDownsampling to AVIF_CHROMA_DOWNSAMPLING_BEST_QUALITY (for avifImageRGBToYUV), and set avoidLibYUV to AVIF_TRUE.

But this is getting a little verbose.